src/applications/disability-benefits/all-claims/pages/index.js

```diff
@@ -1,755 +1,171 @@
-  uiSchema as summaryOfEvidenceUISchema,
-  schema as summaryOfEvidenceSchema,
-} from './summaryOfEvidence';
-
-import {
-  uiSchema as contactInformationUISchema,
-  schema as contactInformationSchema,
-} from './contactInformation';
-
-import {
-  uiSchema as paymentInformationUISchema,
-  schema as paymentInformationSchema,
-} from './paymentInformation';
-
-import {
-  uiSchema as claimExamsInfoUIschema,
-  schema as claimExamsInfoSchema,
-} from './claimExamsInfo';
-
-import {
-  uiSchema as homelessOrAtRiskUISchema,
-  schema as homelessOrAtRiskSchema,
-} from './homelessOrAtRisk';
-
-import {
-  uiSchema as vaEmployeeUISchema,
-  schema as vaEmployeeSchema,
-} from './vaEmployee';
-
-import {
-  uiSchema as fullyDevelopedClaimUiSchema,
-  schema as fullyDevelopedClaimSchema,
-} from './fullyDevelopedClaim';
-
-import {
-  uiSchema as unemployabilityFormIntroUISchema,
-  schema as unemployabilityFormIntroSchema,
-} from './unemployabilityFormIntro';
-
-import {
-  uiSchema as unemployabilityDisabilitiesUISchema,
-  schema as unemployabilityDisabilitiesSchema,
-} from './unemployabilityDisabilities';
-
-import {
-  uiSchema as workBehaviorChangesUISchema,
-  schema as workBehaviorChangesSchema,
-} from './workBehaviorChanges';
-
-import {
-  uiSchema as socialBehaviorChangesUISchema,
-  schema as socialBehaviorChangesSchema,
-} from './socialBehaviorChanges';
-
-import {
-  uiSchema as additionalBehaviorChangesUISchema,
-  schema as additionalBehaviorChangesSchema,
-} from './additionalBehaviorChanges';
-
-import {
-  uiSchema as incidentDescriptionUISchema,
-  schema as incidentDescriptionSchema,
-} from './incidentDescription';
-
-import {
-  uiSchema as secondaryIncidentDescriptionUISchema,
-  schema as secondaryIncidentDescriptionSchema,
-} from './secondaryIncidentDescription';
-
-import {
-  uiSchema as secondaryOtherSourcesUISchema,
-  schema as secondaryOtherSourcesSchema,
-} from './secondaryOtherSources';
-
-import {
-  uiSchema as secondaryOtherSourcesHelpUISchema,
-  schema as secondaryOtherSourcesHelpSchema,
-} from './secondaryOtherSourcesHelp';
-
-import {
-  uiSchema as mentalHealthChangesUISchema,
-  schema as mentalHealthChangesSchema,
-} from './mentalHealthChanges';
-
-import {
-  uiSchema as adaptiveBenefitsUISchema,
-  schema as adaptiveBenefitsSchema,
-} from './adaptiveBenefits';
-
-import {
-  uiSchema as aidAndAttendanceUISchema,
-  schema as aidAndAttendanceSchema,
-} from './aidAndAttendance';
-
-import {
-  uiSchema as individualUnemployabilityUISchema,
-  schema as individualUnemployabilitySchema,
-} from './individualUnemployability';
-
-import {
-  uiSchema as physicalHealthChangesUISchema,
-  schema as physicalHealthChangesSchema,
-} from './physicalHealthChanges';
-
-import {
-  uiSchema as hospitalizationHistoryUISchema,
-  schema as hospitalizationHistorySchema,
-} from './hospitalizationHistory';
-
-import * as medicalCare from './medicalCare';
-
+import * as alternateNames from './alternateNames';
```

Comment:
The changes in this file look like refactoring that's unrelated to the changes you're making, which means they should probably be in a separate PR and discussed with @markgreenburg and @cvalarida.